### ISSUE TITLE
Changes to flags in the entity report - no G2_ENTITY_BRIEF_FORMAT

### ISSUE THREAD

**Issue by jeffadair** [Senzing/resolver#42]
When testing a future version of Senzing, I found this error when running the resolver

```
Traceback (most recent call last):
  File "/app/resolver.py", line 1347, in <module>
    globals()[subcommand_function_name](args)
  File "/app/resolver.py", line 1202, in do_file_input
    result = handle_post_resolver(input_iterator)
  File "/app/resolver.py", line 1127, in handle_post_resolver
    result = g2_client.get_resolved_entities()
  File "/app/resolver.py", line 817, in get_resolved_entities
    flags = G2Engine.G2_EXPORT_INCLUDE_ALL_ENTITIES | G2Engine.G2_ENTITY_BRIEF_FORMAT
AttributeError: type object 'G2Engine' has no attribute 'G2_ENTITY_BRIEF_FORMAT'
```

**Comment by jeffadair** [comment]
These new flags should provide the same info in the report, plus a little bit more.
G2_ENTITY_INCLUDE_RECORD_MATCHING_INFO G2_EXPORT_INCLUDE_ALL_RELATIONSHIPS G2_ENTITY_INCLUDE_RELATED_MATCHING_INFO

**Comment by jeffadair** [comment]
Fixed in branch https://github.com/Senzing/resolver/tree/issue42.jadair.1

**Comment by jeffadair** [comment]
Fixed by #43
